File: src/main/java/org/motivepick/domain/entity/LoginStateEntity.java
package org.motivepick.domain.entity;

import javax.persistence.Column;
import javax.persistence.Entity;

@Entity(name = "LOGIN_STATE")
public
class LoginStateEntity extends AbstractEntity {
    @Column(nullable = false)
    String stateUuid;

    @Column(nullable = false)
    boolean mobile;
}

File: src/main/kotlin/org/motivepick/domain/entity/LoginStateEntity.kt
package org.motivepick.domain.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity

@Entity(name = "LOGIN_STATE")
class LoginStateEntity(
    @Column(nullable = false)
    var stateUuid: String,

    @Column(nullable = false)
    var mobile: Boolean
) : AbstractEntity() {

    constructor() : this("", false)
}
